app/templates/search/agenda-items.hbs

```diff
@@ -1,57 +1,79 @@
-{{ page-title
-  (t "agendaitems")
-}}
-<div class="auk-o-grid auk-u-mb-2">
-  <div class="auk-o-grid-col-2">
-    <div class="auk-box">
-      <div class="auk-u-mb-2">
-        <Auk::Label for="sort-type">
-          {{t "sort-by"}}
-        </Auk::Label>
-        <select
-          data-test-route-search-agendaitems-sidebar-sort-options
-          name="sort-type"
-          id="sort-type"
-          class="auk-select auk-select--block"
-          {{on "input" this.selectSort}}
-        >
-          {{#each this.sortOptions as |sortType|}}
-            <option
-              value={{sortType.value}}
-              selected={{eq sortType.value this.sort}}>
-              {{sortType.label}}
-            </option>
-          {{/each}}
-        </select>
-      </div>
-      <div class="auk-u-mb-2">
-        <Auk::Label>{{t "type"}}</Auk::Label>
-        <Auk::CheckboxList>
-          <AuControlCheckbox
-            @label={{t "note"}}
-            @checked={{this.includeNotas}}
-            @onChange={{this.toggleIncludeNotas}}
+{{page-title (t "agendaitems")}}
+<AuMainContainer as |m|>
+  <m.sidebar>
+    <div class="au-c-sidebar">
+      <div class="au-c-sidebar__content">
+        <Search::Navigation />
+        <div class="au-o-box au-o-box--small au-o-flow au-o-flow--tiny">
+          <Search::DateRangeFilter 
+            @onDateFromChange={{this.onDateFromChange}}
+            @onDateToChange={{this.onDateToChange}}
+            @onDateToClear={{set this "dateTo" undefined}}
+            @onDateFromClear={{set this "dateFrom" undefined}}
+            @dateFrom={{this.dateFrom}}
+            @dateTo={{this.dateTo}}
           />
-          <AuControlCheckbox
-            @label={{capitalize (t "announcement")}}
-            @checked={{this.includeMededelingen}}
-            @onChange={{this.toggleIncludeMededelingen}}
-          />
-        </Auk::CheckboxList>
-      </div>
-      <div>
-        <Auk::Label>{{t "version"}}</Auk::Label>
-        <AuControlCheckbox
-          @label={{t "only-on-final-agenda"}}
-          @checked={{this.latestOnly}}
-          @onChange={{this.toggleLatestOnly}}
-        />
+          <Search::MinisterFilter />
+          <div class="au-c-card au-o-box au-o-box--small">
+            <Auk::Label>{{t "type"}}</Auk::Label>
+            <Auk::Checkbox
```

Comment:
You will have to convert the `Auk::Checkbox` component instances inside your changes to the version from `ember-appuniversum`: `AuControlCheckbox`. You can either check the code (of this page) before your changes or visit the following docs: https://appuniversum.github.io/ember-appuniversum/?path=/story/components-forms-aucheckbox--component.

FYI: the removal of the `AuCheckbox` component and its styling is being executed this sprint: https://github.com/kanselarij-vlaanderen/frontend-kaleidos/pull/1610